cmd/kops/create_cluster.go

```diff
@@ -1203,3 +1248,34 @@ func setOverrides(overrides []string, cluster *api.Cluster, instanceGroups []*ap
 	}
 	return nil
 }
+
+func getZoneToSubnetProviderID(VPCID string, region string, subnetIDs []string) (map[string]string, error) {
+	res := make(map[string]string)
+	cloudTags := map[string]string{}
```

Comment:
Nit: (don't bother fixing) it is odd to mix both ways of building a map in subsequent lines - I think they do the same thing :-)